### PR TITLE
ROX-18275: add central_id to `rox_sensor_info`

### PR DESCRIPTION
**Pull request by stehessel**
## Description

We want Sensor to know the ID of its Central for telemetry purposes. Hence, add the Central ID to the SensorHello message and make it available in `rox_sensor_info`.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Comment by stehessel** [comment]
Current dependencies on/for this PR:
* master
  * **PR #7165** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/7165" target="_blank">Graphite</a> 
    * **PR #7166** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/7166" target="_blank">Graphite</a> 
      * **PR #7167** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/7167" target="_blank">Graphite</a> 
        * **PR #7168** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/7168" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/7168?utm_source=stack-comment).

**Comment by stehessel** [comment]
@connorgorman Adding you as additional reviewer because you touched the CentralHello last and can hopefully confirm we are not breaking anything 😅

File: central/metrics/info/gauge.go [hunk 1]
```diff
@@ -44,7 +45,7 @@ func newGaugeVec() *prometheus.GaugeVec {
 			Name:      "info",
 			Help:      "A metric with a constant '1' value labeled by information identifying the Central installation",
 			ConstLabels: prometheus.Labels{
-				"central_id":      fetchInstallInfo().GetId(),
```
Comment (connorgorman) [inline central/metrics/info/gauge.go:47]: Ideally, would be nice to return an error from the fetch install info function and make it clear that central_id will be empty on error. Also, what does that do to the data we're getting in telemetry

**Review by connorgorman — COMMENTED** [review]
Couple of nits

File: central/metrics/info/gauge.go (hunk 1 above)
Comment (stehessel) [reply central/metrics/info/gauge.go:47]: If `central_id` is empty, then the corresponding labels in `rox_central_info` / `rox_sensor_info` would also be empty, and Prometheus would drop them. On the Telemeter backend the metric series would still be distinguished by the cluster ID of the OpenShift cluster.

**Review by connorgorman — APPROVED** [review]

**Pull request merged**